extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -55,6 +56,12 @@ export class BaseSlides extends BaseCarousel {
     this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
 
     if (this.shouldAutoplay_) {
+      this.registerAction('toggleAutoPlay', invocation => {
+        const args = invocation.args;
+        if (args) {
+          this.toggleAutoPlay(args['toggleOn']);
+        }
+      }, ActionTrust.HIGH);
```

Comment:
`.LOW` should be fine for this. Similar actions such as playing a video or animation is low trust as well since they are sandboxed UI-wised.

Reply:
Done